rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/function/FunctionGenerator.xtend

```diff
@@ -284,7 +285,7 @@ class FunctionGenerator {
 							«doEvaluateScope.getIdentifierOrThrow(input)» = «Collections».emptyList();
 						}
 						«ENDFOR»
-						«output.toBuilderType» «doEvaluateScope.getIdentifierOrThrow(output)» = «IF output.multi»new «ArrayList»<>()«ELSEIF output.needsBuilder»«output.RType.toListOrSingleJavaType(output.multi)».builder()«ELSE»null«ENDIF»;
```

Comment:
This is another interesting one.
```
func A:
  output:
    result string (1..1)
      [metadata scheme]

func B:
  output:
    result string (1..1)
  set result:
    A() -> scheme
```
Does this generate valid Java code?

Reply:
See test `testCanPassMetaFromOutputOfFunctionCall`